Formula/idris2.rb

```diff
@@ -15,7 +15,8 @@ class Idris2 < Formula
 
   depends_on "chezscheme"
   depends_on "coreutils"
-  depends_on "gmp"
+  depends_on "gmp" => :build
```

Comment:
You need to put this before all the other dependencies. When you fix this, please squash all your commits together with your original commit. Thanks!

Reply:
Let me know if you need any help with this.

Reply:
It's been a while since I've done anything with Homebrew (or Ruby). I'm running `brew audit` locally now. Thanks for all the help.